optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingCfpDevoxxImporter.java

```diff
@@ -259,7 +262,7 @@ private void importTalkList() {
         Set<String> themeTrackSet = new HashSet<>(Arrays.asList(talkObject.getJsonObject("track").getString("id")));
         if (!trackIdSet.containsAll(themeTrackSet)) {
             throw new IllegalStateException("The talk (" + title + ") with id (" + code
-                    + ") contains trackId + (" + trackIdSet + ") that doesn't exist in the trackIdSet.");
+                    + ") contains trackId " + themeTrackSet.toString() + " that doesn't exist in the trackIdSet.");
```

Comment:
toString() is obsolete and use () to delimate start and end of value like in all other exceptions (even despite Set adding [] around it), so
"... trackId (" + themeTrackSet + ") ..."